bin/setup

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+if ! command -v ja > /dev/null; then
+  brew install jq
```

Comment:
I assume this locks us into macOS? I don't think we should worry about it right now, but something to think about.

Reply:
Yeah, `jq` itself is very portable so other OS's could install it. I just don't know how to put that into this script 🙈

Reply:
Probably React, right?
<br>
<br>
<br>
<br>
<br>
<br>
<br>
:trollface: